### PR TITLE
Equality

### PR DESCRIPTION
**Pull request by smdern**

### CODE REVIEW

File: test/automated/equality/not_equal.rb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        other_collection = Collection::Set.new(String)
+
+        not_equal = collection != other_collection
+
+        test "Is not equal" do
+          assert(not_equal)
+        end
+      end
+
+      context "Different set type" do
```
Comment (sbellware) [inline test/automated/equality/not_equal.rb]: @smdern Could you rename this context to "Different Type Parameter"?

File: test/automated/equality/not_equal.rb [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        other_collection = Collection::Set.new(Integer)
+
+        not_equal = collection != other_collection
+
+        test "Is not equal" do
+          assert(not_equal)
+        end
+      end
+
+      context "Different entries" do
```
Comment (sbellware) [inline test/automated/equality/not_equal.rb]: Contexts should be title case, eg: "Different Entries"

**Comment by smdern** [comment]
@sbellware Context titles have been addressed.

**Comment by sbellware** [comment]
Thanks, @smdern

**Pull request merged**